examples/flutter_gallery/lib/gallery/theme.dart

```diff
@@ -12,7 +12,7 @@ class GalleryTheme {
 }
 
 const MaterialColor _kPurpleSwatch = const MaterialColor(
-  500,
+  0xFF6200EE,
```

Comment:
Whoa.  I guess any int will do. :-)

Reply:
I should fix that with a constant. It's just the value of the 500 entry.

Reply:
Yeah, a constant seems like a good idea.